Sources/muterCore/Extensions/TaskExtensions.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension Task where Success == Never, Failure == Never {
```

Comment:
What's this used for? A task that doesn't produce a result if it succeeds or fails is...interesting to consider. :joy:

Reply:
I've removed it but it was just simpler way to call `Task.sleep` hiding the math behind it :)